<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevList</class>
 <widget class="QDialog" name="DevList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Noise Filter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="nf_resources.qrc">
    <normaloff>:/icons/noise_filter_48</normaloff>:/icons/noise_filter_48</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="devListGroup">
     <property name="title">
      <string>Device list</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTreeWidget" name="devList">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scanButton">
        <property name="text">
         <string>Scan</string>
        </property>
        <property name="icon">
         <iconset resource="nf_resources.qrc">
          <normaloff>:/icons/scan_48</normaloff>:/icons/scan_48</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="blockableListGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Blockable items</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QListWidget" name="blockableList">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="blockable"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="nf_resources.qrc">
            <normaloff>:/icons/add_48</normaloff>:/icons/add_48</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="nf_resources.qrc">
            <normaloff>:/icons/remove_48</normaloff>:/icons/remove_48</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="gridLayout"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="nf_resources.qrc"/>
 </resources>
 <connections/>
</ui>
